.changeset/modern-geckos-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@razorpay/blade': patch
+---
+
+fix: fileupload on remove clashes with onchage
```

Comment:
```suggestion
fix(FileUpload): fileupload on remove clashes with onchage
```


Also can you add a note here for consumers on what exactly we changed in behaviour?

Reply:
@saurabhdaware note added